server/pkg/api/user.go

```diff
@@ -485,6 +487,7 @@ func (h *UserHandler) GetAccountsToken(c *gin.Context) {
 	}
 	c.JSON(http.StatusOK, gin.H{
 		"accountsToken": token,
+		"accountUrl":    viper.GetString("apps.account"),
```

Comment:
Typo, I think, the config key is "apps.accounts" (s). While we're at it, can we also change the JSON accountUrl => accountsUrl - no functional impact, just for consistency (familyUrl is fine, don't need to change that, just changing  accounts would be good since the rest of the PR uses the plural)